tox.ini

```diff
@@ -118,63 +118,85 @@ basepython = python2.6
 
 [testenv:py3.5-smoke-tests]
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME
 commands =
-    py.test smoke_tests/standalone_smoke_tests -s -vv --durations=5
+    py.test tests/smoke_tests/standalone_test.py -s -vv --durations=5
 
 [testenv:py3.6-smoke-tests]
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME
 commands =
-    py.test smoke_tests/standalone_smoke_tests -s -vv --durations=5
+    py.test tests/smoke_tests/standalone_test.py -s -vv --durations=5
 
 [testenv:py3.7-smoke-tests]
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME
 commands =
-    py.test smoke_tests/standalone_smoke_tests -s -vv --durations=5
+    py.test tests/smoke_tests/standalone_test.py -s -vv --durations=5
 
 [testenv:py3.8-smoke-tests]
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME
 commands =
-    py.test smoke_tests/standalone_smoke_tests -s -vv --durations=5
+    py.test tests/smoke_tests/standalone_test.py -s -vv --durations=5
 
 # Package smoke tests related targets
 [testenv:agent_package_smoke_test_amazonlinux_python2]
 basepython = python3.6
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME DOCKER_CERT_PATH DOCKER_HOST DOCKER_TLS_VERIFY
 commands =
-    py.test smoke_tests/package_smoke_tests -s -vv --durations=5 --package-distribution amazonlinux --package-python-version python2 {posargs}
+    py.test tests/smoke_tests/package_test.py::test_smoke_package_rpm_python2 -s -vv --durations=5 {posargs}
 
 [testenv:agent_package_smoke_test_amazonlinux_python3]
 basepython = python3.6
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME DOCKER_CERT_PATH DOCKER_HOST DOCKER_TLS_VERIFY
 commands =
-    py.test smoke_tests/package_smoke_tests -s -vv --durations=5 --package-distribution amazonlinux --package-python-version python3 {posargs}
+    py.test tests/smoke_tests/package_test.py::test_smoke_package_rpm_python3 -s -vv --durations=5 {posargs}
 
 [testenv:agent_package_smoke_test_ubuntu_python2]
 basepython = python3.6
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME DOCKER_CERT_PATH DOCKER_HOST DOCKER_TLS_VERIFY
 commands =
-    py.test smoke_tests/package_smoke_tests -s -vv --durations=5 --package-distribution ubuntu --package-python-version python2 {posargs}
+    py.test tests/smoke_tests/package_test.py::test_smoke_package_deb_python2 -s -vv --durations=5 {posargs}
 
 [testenv:agent_package_smoke_test_ubuntu_python3]
 basepython = python3.6
 passenv =
     SCALYR_API_KEY READ_API_KEY SCALYR_SERVER AGENT_HOST_NAME DOCKER_CERT_PATH DOCKER_HOST DOCKER_TLS_VERIFY
 commands =
-    py.test smoke_tests/package_smoke_tests -s -vv --durations=5 --package-distribution ubuntu --package-python-version python3 {posargs}
+    py.test tests/smoke_tests/package_test.py::test_smoke_package_deb_python3 -s -vv --durations=5 {posargs}
+
+
+[testenv:agent_distributions_tests_ubuntu]
```

Comment:
How do we plan to run those on Circle CI?

I didn't see we hooked up those targets in circle ci config yet. Or did I miss it?

EDIT: It seems like we run it as part of the existing package test job?

Reply:
Yes, those tests are not in CircleCI yet. In theory, all those tests can be executed in one job, or we can split them into one job per each OS distro. I think we need to decide what would cost less.

Reply:
We already have tons of separate builders so maybe not a bad idea to combine some of those together (this way we also avoid some overhead of spinning up new Circle CI environment for each job).

It mean mean though that everything will take slightly longer to finish due to those jobs not running concurrently...

But we can also experiment and see, in theory combining it should be slightly cheaper because we avoid overhead of spinning up the environment (which doesn't take that long), but it means the whole build will take longer to complete.